### DISCUSSION TITLE
Does it work on G7 7790?

### DISCUSSION THREAD

**Discussion by alexmurari** [T-Troll/alienfx-tools#178] [Q&A]
This is the status result: 

![image](https://user-images.githubusercontent.com/11204378/185680946-30a74b11-2661-4f50-80f4-6023a9b8a97e.png)

When trying to execute `lightson`, it does nothing.

Btw, i'm really glad that there is an alternative to the crappy AWCC. For some reason the new version was leaking memory....

**Comment by T-Troll** [comment]
> Device #0....

Yes, your chip is supported.

You only have 1 light defined. lightson/lightsoff use light list, so now it can control this light only.
You can play with hardware lights directly (using their ID) by `set-one` command. 

Please keep in mind - light IDs different between different models, some of them do nothing.
Are you sure you define it correctly? Please do it using `probe` command or GUI!

**Reply by alexmurari** [reply]
Thanks, @T-Troll!

I'm using the `probe` command and trying to get a handle of it. It's tricky hehehehe.

Meanwhile, I've tried to use the GUI, but the app opens on the following screen, freezes for 2~3 seconds and exits... what could it be? I can't press anything, it just freezes and exits.

![image](https://user-images.githubusercontent.com/11204378/185751375-6d88311b-1276-4a47-8994-4faeba6fe996.png)

**Reply by alexmurari** [reply]
Using the `probe` command, It only identified the following zones in my keyboard.

![IMG_20220820_113632](https://user-images.githubusercontent.com/11204378/185751828-fe11d184-ee92-4f28-8ddc-f67a4c79fc63.jpg)

But it has probed a lot of devices (the lights (zones) that turned on are `#0` and `#1` from `#2` onwards, it did nothing)...

![image](https://user-images.githubusercontent.com/11204378/185751934-4abc8f17-1adb-45c9-8125-7d36a83eae05.png)

**Comment by T-Troll** [comment]
> Meanwhile, I've tried to use the GUI, but the app opens on the following screen, freezes for 2~3 seconds and exits... what could it be? I can't press anything, it just freezes and exits.

Ohshi... It's a bug, can you please open an issue - it needs to be resolved!

**Comment by T-Troll** [comment]
> But it has probed a lot of devices (the lights (zones) that turned on are #0 and #1 from #2 onwards, it did nothing)...

So no reaction for any ID... Interesting. Let me check DB, in fact it's not 22 of them, but 255.

**Reply by alexmurari** [reply]
@T-Troll I've opened the issue.

It's really weird that it's only showing 22 ids...

**Reply by T-Troll** [reply]
Check `alienfx-cli` usage, you can change how many IDs it will scan. 22 is default for non-keyboard.

**Reply by alexmurari** [reply]
@T-Troll I can't make the probe command accept the lights (`l`) argument. I've tried `alienfx-cli.exe probe -l 255` .... `alienfx-cli.exe probe l=255`... `alienfx-cli.exe probe l 255`... but none identified the `l` argument.

**Reply by T-Troll** [reply]
Look at usage - argument format is different!
Correct is `probe=l,255`.

**Reply by T-Troll** [reply]
Uhhh... First, read the beginners manual - this wrote BIG LETTERS at the top of EVERY release. You miss a point in GUI completely!
I'll check CLI as well, but seems like you miss light changes as well.
